README.md

```diff
@@ -440,12 +385,55 @@ Collect all external CSS and append a
     ...
   }
 ];
-</code></pre><p>Next, run the script with <code>--steps=/path/to/mysteps.js</code>:</p>
-<pre><code># Amplify a page with customized steps.
+```
+
+Next, run the script with `--steps=/path/to/mysteps.js`
+
+```
+# Amplify a page with customized steps.
 ./amp-prototyper http://127.0.0.1:8080 --steps=/path/to/mysteps.js
-</code></pre><h2 id="reference">Reference</h2>
-<ul>
-<li><a href="https://github.com/GoogleChrome/puppeteer">puppeteer</a></li>
-<li><a href="https://github.com/jakubpawlowicz/clean-css">clean-css</a></li>
-<li><a href="https://github.com/purifycss/purifycss">purifycss</a></li>
-</ul>
+```
+
+## Use cases with specific conditions
+
+### Sites with Crawler Protection
+
+Since AMP-prototyper is built with [Puppeteer](https://github.com/GoogleChrome/puppeteer) to fetch a page content, some sites may treat this as a crawler or bot, and thus block the access. You may see a different content (e.g. 404 page or timeout) than the regular page content.
+
+The blocking logic usually resides in the web hosting servers. Hence, there's no easy way to overcome. However, you can try the following options:
+
+#### Option 1 - Set headless=false
+
+Some sites may treat puppeteer activities as crawler if it runs in the headless mode. You can try running with headless off, and puppeteer will show the browser window like browsing by a real user.
+
+```
+# Amplify a page without headless mode.
+./amp-prototyper http://127.0.0.1:8080 --headless=false
+```
+
+#### Option 2 - Serve the page locally
+
+Another option to bypass crawler-protection is to download the page and serve the page from a local web server. (E.g. [http-server](https://www.npmjs.com/package/http-server))
+
+For example, you can save the index page of [ThinkWithGoogle.com](https://www.thinkwithgoogle.com) and serve it locally.  It's recommended to copy the **page source** directly instead of using a browser's Save function to avoid unexpected artifacts by the browser.
+
+Since the host is now `localhost` instead of its original host (www.thinkwithgoogle.com), all relative asset paths would incorrectly end up with localhost. Hence, we need to pass `--customHost` to restore its true remote host for asset URLs.
+
+```
+# Assuming that the index page is saved to ./tmp/web/index.html
+
+npm i http-server -g
+http-server ./tmp/web -p 3000
+```
+
+In a new terminal:
+
+```
+./amp-prototyper http://127.0.0.1:3000 --customHost=https://www.thinkwithgoogle.com
```

Comment:
Have you tested to see this is working as expected if running as an npm global install?